packages/shared/src/lib/featureManagement.ts

```diff
@@ -19,6 +19,7 @@ export class Feature<T extends JSONValue> {
 }
 
 const feature = {
+  feedActionSpacing: new Feature('feed_action_spacing', false),
```

Comment:
Can you extract at bottom as seperate export please 🙏

Reply:
Ah, yes. I'll move it quickly.